src/Directory.Build.props

```diff
@@ -5,9 +5,11 @@
     <Nullable>enable</Nullable>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <!-- NU5104: A stable release of a package should not have a prerelease dependency -->
-    <NoWarn>$(NoWarn);NU5104</NoWarn>
+    <NoWarn>$(NoWarn);NU5104;CS1573;CS1591</NoWarn>
```

Comment:
Since we're now using `.globalconfig`, would it make sense to add these suppressions there instead?
e.g. `dotnet_diagnostic.CS1573.severity = silent`

Reply:
I've addressed this